### PR TITLE
Make description as optional parameter in jira module

### LINKED ISSUE #46022: JIRA Module required params bug
##### SUMMARY
Description field is not available on issue create screen. Description is required for module to run create operation.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
jira

##### ANSIBLE VERSION

```paste below
ansible 2.7.0.a1.post0
  config file = /home/repos/Ansible/playbooks/ansible.cfg
  configured module search path = ['/home/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.6/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.6.5 (default, Mar 29 2018, 18:20:46) [GCC 8.0.1 20180317 (Red Hat 8.0.1-0.19)]

```
##### OS / ENVIRONMENT
Fdora 28

##### STEPS TO REPRODUCE
Create an issue, disable the "description" field on the create screen. 
Try to create a ticket with below jira syntax.

```yaml
  - name: Create theTicket
    jira:
      timeout: 20
      uri: '{{ jira_server }}'
      username: '{{ jira_username }}'
      password: '{{ jira_password }}'
      validate_certs: false
      issuetype: "System Request"
      project: proj
      operation: create
      summary: "test"
```

##### EXPECTED RESULTS
I want to be able to create the issue even if description is not enabled.

##### ACTUAL RESULTS
Without description field set.
```
Operation create require the following missing parameters: description
```

Adding Description field returns 
```
HTTP Error 400: 
```

Debugging further shows the 400 returns the below
```
{"errorMessages":[],"errors":{"description":"Field 'description' cannot be set. It is not on the appropriate screen, or unknown."}}
```

### PR DESCRIPTION
**Pull request by stobias123**
##### SUMMARY

Fixes: https://github.com/ansible/ansible/issues/46022

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

jira
##### ANSIBLE VERSION

Only tested below.
```paste below
ansible 2.7.0.a1.post0
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/web_infrastructure/jira.py [hunk 1]
```diff
@@ -369,7 +371,7 @@ def link(restbase, user, passwd, params):
 
 
 # Some parameters are required depending on the operation:
-OP_REQUIRED = dict(create=['project', 'issuetype', 'summary', 'description'],
+OP_REQUIRED = dict(create=['project', 'issuetype', 'summary'],
```
Comment (Akasurde) [inline lib/ansible/modules/web_infrastructure/jira.py:375]: Could you please add a note in DOCUMENTATION section stating this requirement ? Thanks.

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by gundalow** [comment]
Could you please add a changelog/fragment for this, see https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs

File: lib/ansible/modules/web_infrastructure/jira.py (hunk 1 above)
Comment (webknjaz) [reply lib/ansible/modules/web_infrastructure/jira.py:375]: (and a changelog note)

Comment (stobias123) [reply lib/ansible/modules/web_infrastructure/jira.py:375]: - added changelog 
- documentation already shows that description is not required [L#59](https://github.com/ansible/ansible/blob/aabc1e072c59bd9074613f2fc6e9eab2109cab75/lib/ansible/modules/web_infrastructure/jira.py#L59)

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged into `devel` for release in Ansible 2.8, thanks

**Pull request merged**